patterns/card-with-hyperlink-list.php

```diff
@@ -0,0 +1,301 @@
13lGDbtiBSyqc6SohtgUHK9c2OEmr7LZiUJZccJcw+D+a0ve8otew5kiJ3p7aj1LZ97UdK28aOUse2LYCUkuaOUtf2gZuUqlhHqWdbEEJKbe0o9d1Fv87vDwX7zEh1lHB7n1BSrndzlEfcxW8mNoGiSDmR6SiP2h51kbK91FEeszMEFhwa4yiP11igDCSxECVD0rKmF5dMmiQFFbHx/xUj5IcBC15x1ffLLbi3I3DBw8a7q242vNmsS50LXZpS05hBlWsEfkjlERQhOLORw5StyEtkIGXUbPq/laIXSLGNRk4QH058KuFmxA5RQjzgefgY3oTf4CH4AqfG+XASVCU1CKPEMOZQje0wxE5DjXAZxw0xSIoJ0qhuRME1terrKehz3FaGig0xESCNZBPRnBaTA59s0aKoT+/exVGRkRBc+KAeaISTw4228fFvxMZ54+K9WfG92vaKebPDG73ax73Rq8PT8e3i4jMz4zJ7tY+NeTpTyNi4+PiEmA40WUiXSwlxxXCFGQ1Ea3nAiF4TLNxhHYVxbeBDTQOEvP7FNpPQR1Zsb2/JawMtbZPgg7NMIg8dQmOFuGUIo0dYqPWP1UO1ArnSLU6PyPG3Or1+1jy857Z+wl1qqqlbTuhOr2clxQ3dSPObrl3YcDKjog8mi5/1mu1fW20CeSmQRhlndOGYx/S8bcHWmTlTGBWtywR7YQAQM6hZlIZrMAVSkF1HX//iGd5+b9QURjUxbx+TJoMv8wAgzwAZtGGHxoAH82/o9dMnM6qdM7swwZIYAMQMsYjO3WTwV2yKKt+0gsmfQzbp8SnB1vxtO3V/M8iSx5pt06Hyslaz+32te4Zd1ZeaV+hPJ1zTFfnzGFXu+Q+YXGrejhauZJyYN1pLEPKvkB/UX+SLC0jXDoGWpELSTEJlSz2s4wktZ0S0YaK8Z9N05du/6lWNB7NFNSaKUgkHjeMv6sycI0zupLust948p4FwsASpV1aiJc4JouD/VY39tA7qSTjY/O/znzOqVY23Mmm99Kh+Z910it497T1bZUqQ3bt2Jank2GO6b92yhJ0zt3KMh3WcwajOVUUxQYxjAhZw0GXLj31BR6pMnvQ421djsDB+ejBWRy25qVXV7XQm8IF/6QGdNvJR3ShjtpbdRh5konTQQf3VjX361VGbeGTNVLGdXAAfki9/48x31jXxObNnWCsOkazVKoYJ79X2qLLoNmUzLtyfz6h+erCfyaXmN/TYx5S18scH+tt3L2o5rKOwQNSBdwMpO5W1sGGEhZFrUp9iVONT/sJk/fQM6+H+O5RpGVbkkqVagpBhQjbK6OyQQ7OWaAVyK/p7vTcq0UIyVuRHM6qb+0KZIGxIXWDayAm+ONISQi5seNiXRSC8GAgWc7X8D2f2xh0zGelQ3Zkg8fx67KnGT38nE+n5NI3SH2kRuCDJour2MKnzt3hN5c/o16O1pUAaZTSzMEAIj4UZQPXod3toJY+FxMEAIM8AwRIYgCUxA+iLHMcJ5tiEjHHsYgJJrKsOdfmmwnX2r07d2KHGEiBqyOVulDDSihtaypGlOYwKHgFp8UotqyL/ggaiASiQrZ4lJgYgmXt/tpBRUcozQSmiDrDH+JThJlcriIoJeAp+MqFQN02sxrVsjm0yiDDwk90PeGOQNakh+nHVg5vZexfnMlKpHmKy1XOdKwjY3xymFUjXDpXUSw9xhLufnsOobkW/wgQZj/zB0thMVJL/UL6oLq7jl+OfNp/51zIyk8uWUZ16vz6TyrfXmXOndE6kDCLckiRBqEZMRTVoUtWYqPhmR94z76ZnMqqsxOlMkPolfdYykke1AiHf0G0xh9pJmk6OfQ0DtRqRc94syP4MP5rIWSBlCbc8c06/B1q29OdIkQsYFVoVyLWB6/U/f/C3cHzEWMIXi589qVWtVku5PReOmcgtAs0YSEIME2ri+qXTZxkLsvdqBUL3lP6gfmsup3nbhusGbaZpZYgNHB4469CsfozwnthAlUL2mrP7mVqCcAYiBWzy0bgUzUnxySPfkd+6cxS/vDGaUSUd/4ZJ8MHTfEZg6bgyLUF4jYr8AOpR1YTqQNCqV3ylgduSvGbKu+mnmSgcg1Iradm0xbRU07VrUkcw2tGEjQnLn4OT2WhxkJJh967qUbg2QSgJOLLAhOWf+Hr3+xfPUf7UJa8EWydeGMmoDu9ZyQQbn6uS1v7nT+kZET9otg2E+tsZemfc0lDwdABihgR5dVQnS4Yv8rdsQucSVEyXfSfE5K8HjDF5MAjV8l6KydEkbAxTa2wXxje3x7ArqXHuZnKu6hRWIbuAdO+B/KkdHQfkXvGXWEhTTmssiptF0etIb/75gv5lfgXVzU1sychPBRBcnQnLtzOGL6JbDQQFiwHYtMYK0cZX3nwwELLE18MIfcSWQMQRSiUUy7Jp5D1qMxTDasJpXUll2GQwua1vXRdZEG1JFJJNOEPwHrAJT4dkk407IrUCwVroDit/LOKrGUg3xFgmbY/SfdvyYxMtj65oU4KQkxLpOTeLSOc1ScdHscu/v1vMqC7cb8EE5lNnSALi5pUg2F5soT+m4UucQD318bdMyjf9rJ87eY8SFrhaSxByjqAWs99H1qTu0vR2MWV/cwcTRbbT2yXZRITQgPEqoP6VRC+DCWwdPVvYGBwQaFv3hNvAgMR6ZRvocFeTFE7boM2rJv0BaeJHIF5ECoQcT+8bF7WSmXQ1T6EVZmmJNxx6FxO8CxUIqq/IWsE9xXu2nFFV5GczIdvoSr9ALvXXeE1JyjMTD2x6pzXQx5q5rWEdX2d31PhHGGhh+6L+rrmXHWs2RNPlNoNsH6TV6KsDyawyfXPf89zQfMtna4U7Ad0Fudj+tQRGtCgF8nL8cvPzLu3Ra030LyDdcaOYoJPSu8JEv6RXsKnwjqdHD/WySrpNnqCU8pkjxP8A(/figma)-->"></span>Lorem ipsum dolor sit ametIpsum Lorem</a></li>
```

Comment:
There are some monster comments in here that seem to come from figma. Do we know what these do?